app/src/main/res/layout/external_player_fragment.xml

```diff
@@ -27,7 +27,8 @@
                 android:layout_height="match_parent"
                 android:adjustViewBounds="true"
                 android:cropToPadding="true"
-                android:scaleType="centerCrop"
+                android:maxWidth="200dp"
```

Comment:
Thanks! 200dp seems to be a pretty big limit. On my screen, this nearly fills the complete width. I think something like 96dp would be better to ensure that the text still has enough space.

Reply:
Updated the size